### PR TITLE
ref(css): Remove global overflow: hidden for modals

### PR DESCRIPTION
**Pull request by macqueen**

### CODE REVIEW

File: src/sentry/static/sentry/less/group-detail.less [hunk 1]
```diff
@@ -2802,3 +2802,9 @@ ul.crumbs {
     display: none;
   }
 }
+
+.issue-modal {
+  .modal-content {
+    overflow: visible;
+  }
+}
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/less/group-detail.less]: 😬 will let @Chrissy or @billyvg decide if we want to just remove it globally or not

Comment (billyvg) [reply src/sentry/static/sentry/less/group-detail.less]: I think we should remove globally - I'd rather avoid this problem for future features than worry about any potential regressions.

Comment (macqueen) [reply src/sentry/static/sentry/less/group-detail.less]: ok, updated!

Comment (Chrissy) [reply src/sentry/static/sentry/less/group-detail.less]: agree with billy.

there is a decent chance this was used as a clearfix, in which case you could do `overflow: auto` but that might inject random scrollbars into places so probably just remove it.

File: src/sentry/static/sentry/less/shared-components.less [hunk 2]
```diff
@@ -2743,12 +2742,6 @@ ul.radio-inputs {
   }
 }
 
-.custom-resolution-modal {
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/less/shared-components.less]: Are these classes used anywhere?

Comment (macqueen) [reply src/sentry/static/sentry/less/shared-components.less]: not anymore :)

**Review by evanpurkhiser — APPROVED** [review]

**Comment by billyvg** [comment]
THANK YOU!

**Pull request merged**